test/hotspot/jtreg/ProblemList-Virtual.txt

```diff
@@ -31,6 +31,8 @@ serviceability/dcmd/thread/PrintConcurrentLocksTest.java 8308033 generic-all
 serviceability/dcmd/thread/PrintTest.java 8308033 generic-all
 serviceability/dcmd/thread/ThreadDumpToFileTest.java 8308033 generic-all
 serviceability/tmtools/jstack/DaemonThreadTest.java 8308033 generic-all
+vmTestbase/nsk/jvmti/scenarios/allocation/AP04/ap04t001/TestDescription.java 8308985 generic-all
+vmTestbase/nsk/jvmti/scenarios/allocation/AP04/ap04t002/TestDescription.java 8308985 generic-all
```

Comment:
I've closed the 8308985 as a dup of:
[8308978](https://bugs.openjdk.org/browse/JDK-8308978) regression with a deadlock involving FollowReferences
Should the 8308985 be replaced with the 8308978?
There is already a tested fix for 8308978.
Alex is going to post a PR today or tomorrow.
Do we still want to problem-list these two tests?

Reply:
Please switch from 8308985 to 8308978. I've also changed this bug
into a sub-task of 8308978 and bumped the priorities of both bugs
to P2. Regressions start at P2 and ProblemListing match the priority
of their parent bug.